docs/src/pages/components/lists/CheckboxList.tsx

```diff
@@ -33,25 +33,25 @@ export default function CheckboxList() {
           <ListItem
             key={value}
             role={undefined}
```

Comment:
Its correct place is on the `ListItemButton`

<img width="890" alt="Capture d’écran 2021-06-06 à 18 31 16" src="https://user-images.githubusercontent.com/3165635/120932309-8d9b8980-c6f5-11eb-8099-aaa833e82dec.png">

https://validator.w3.org/nu/?doc=https%3A%2F%2Fdeploy-preview-26446--material-ui.netlify.app%2Fcomponents%2Flists%2F